azkaban-common/src/main/java/azkaban/project/Project.java

```diff
@@ -39,6 +39,7 @@
   private int version = -1;
   private long createTimestamp;
   private long lastModifiedTimestamp;
+  private String currentUploaderID;
```

Comment:
By current, do you mean the latest uploader of the project ?

How is this information being persisted? Or is this a transient field?
Also I am not sure whether this data is set on all code paths. Since, every project will be uploaded by some user, this should not be an optional data.